ftw/zipextract/browser/templates/zipextract.pt

```diff
@@ -0,0 +1,22 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      i18n:domain="ftw.zipextract"
+      metal:use-macro="context/main_template/macros/master">
+
+    <metal:block fill-slot="header">
+    Extract files
+    </metal:block>
+    <metal:block fill-slot="main">
+        <form tal:attributes="action string:${here/absolute_url}/zipextract">
```

Comment:
I'd use `method="POST"`. You don't want the values to end up in the URL (as it happens with `GET`)